### DISCUSSION TITLE
It's necessary that DS supports to stop and start a task in workflow.

### DISCUSSION THREAD

**Discussion by msj100f** [apache/dolphinscheduler#12442] [Ideas]
In our case ,i have to stop one task in a running workflow, and then start it after optimize some args. I don't want to stop the whole workflow,  because other sub-processes in it are running.

**Comment by DarkAssassinator** [comment]
hi @msj100f , DS has plan to support start a task in workflow in Q4, please check: 
+ https://github.com/apache/dolphinscheduler/discussions/12374

**Reply by msj100f** [reply]
It's really good news, thx.
